Sources/Networking/HTTPClient.swift

```diff
@@ -187,13 +187,20 @@ private extension HTTPClient {
         if let networkError = networkError {
             return Result
                 .failure(networkError)
-                .mapError { error in
+                .mapError { error -> NetworkError in
                     if self.dnsChecker.isBlockedAPIError(networkError),
```

Comment:
I'm thinking it might be worth extracting this whole conversion out.

Reply:
I'm a big fan of that 😄

Reply:
Ugh I forgot I gave myself a comment cause I didn't get an e-mail for it lol
I'll do this in a separate PR.

Reply:
Hah, it seems separate PR worthy anyway

Reply:
#1518